modules/core/shared/src/main/scala/lucuma/core/math/dimensional/unit.scala

```diff
@@ -18,26 +21,32 @@ import coulomb.define._
  */
 trait UnitType { self =>
   type Type
-  def definition: UnitDefinition
```

Comment:
I was going to do matching on `definition` should I use `name` instead?

Reply:
Can you use `===` ?